packages/figma/src/codegen/targets/react/component/handlers/tabs.ts

```
@@ -1,131 +1,14 @@
-  TabsProperties,
-} from "@/codegen/component-properties";
-import { defineComponentHandler } from "@/codegen/core";
-import * as metadata from "@/entities/data/__generated__/component-sets";
-import type { NormalizedInstanceNode } from "@/normalizer";
-import { camelCase } from "change-case";
 import { createLocalSnippetHelper } from "../../element-factories";
-import type { ComponentHandlerDeps } from "../deps.interface";
-import { handleSizeProp } from "../size";
 
-const { createLocalSnippetElement } = createLocalSnippetHelper("tabs");
+const { createLocalSnippetElement: createTabsLocalSnippetElement } =
```

Biome (2.1.2)

[error] This variable createTabsLocalSnippetElement is unused.

Unused variables are often the result of an incomplete refactoring, typos, or other sources of bugs.

(lint/correctness/noUnusedVariables)